### PR TITLE
chore: update dependency rc-tabs to v12 alpha1

### PR DESCRIPTION
**Pull request by tempoeric**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
#36959

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -149,7 +149,7 @@
     "rc-steps": "~4.1.0",
     "rc-switch": "~3.2.0",
     "rc-table": "~7.25.3",
-    "rc-tabs": "~11.16.0",
+    "rc-tabs": "^12.1.0-alpha.1",
```
Comment (afc163) [inline package.json:152]:
```suggestion
    "rc-tabs": "~12.1.0-alpha.1",
```

feature branch please.

Comment (tempoeric) [reply package.json]: already changed and resolved

**Comment by MadCcc** [comment]
Switch base from master to feature plz.

**Comment by tempoeric** [comment]
will create new pr for this

**Comment by tempoeric** [comment]
> Switch base from master to feature plz.

do i need to create new pr ?? or was it merged to feature?

**Comment by afc163** [comment]
You should checkout new branch from feature then send it.

**Pull request closed without merging**